Caesium benchmark

garibay-j opened the pull request:
This adds an OGS-6#Phreeqc model of the Caesium benchmark published in [[1]](https://rdcu.be/cyeMV). Simulation has a large computational time (21h). Therefore, it is not recommended to run the test in the jupyter-notebook. Verification can be done by executing `ogs caesium_phreeqc/caesium.prj -o results`. After this, results can be plotted in the jupyter-notebook without modification:

https://github.com/garibay-j/ogs-jupyter-lab/blob/CsBenchmark/icross-model-chain/CaesiumSorption/ogs-jupyter-lab-icross-caesium-sorption.ipynb

Currently, the test can only be run with the ogs-branch: 

https://gitlab.opengeosys.org/garibay-j/ogs/-/tree/SurfaceMoles

See also relevant merge request:

https://gitlab.opengeosys.org/ogs/ogs/-/merge_requests/3796

Reference:
[1] Águila, J.F., Montoya, V., Samper, J. et al. Modeling cesium migration through Opalinus clay: a benchmark for single- and multi-species sorption-diffusion models. Comput Geosci 25, 1405–1436 (2021). https://doi.org/10.1007/s10596-021-10050-5

OlafKolditz commented:
Dear Jaime, well done! Shall we merge all the results files (VTU)? Best regards, Olaf

garibay-j commented:
> 
> 
> Dear Jaime, well done! Shall we merge all the results files (VTU)? Best regards, Olaf

Thank you Olaf. Good point. I think we can do without the VTU results. I just removed them (also added a .gitignore to keep results locally). 

I also updated a bit the .prj and the dev branch according to comments from the merge request. This may be changed slightly after the merge request is closed.
